pycbc/events/coinc_rate.py

```diff
@@ -0,0 +1,156 @@
+#
+# =============================================================================
+#
+#                                   Preamble
+#
+# =============================================================================
+#
+""" This modules contains functions for calculating and manipulating
+coincident triggers.
+"""
+
+import itertools
+import numpy
+import pycbc.detector
+
+
+def multiifo_noise_coinc_rate(rates, ifos, slop):
+    """
+    Calculate the expected rate of noise coincidences for multiple detectors
+
+    Parameters
+    ----------
+    rates: list of lists/arrays
+        List of vector of single-detector trigger rates
+        in order of template id
+    ifos: list of strings
+        list of ifos in e.g. 'H1' form
```

Comment:
I'm guessing that it needs to get the list of ifos in the same order as the list of trigger rates.  if so, would it work better to use a dictionary as input (ie keyed on the ifo name) ?